src/coordination/azure/Akka.Coordination.Azure/Internal/AzureApiImpl.cs

```diff
@@ -167,6 +167,7 @@ public async Task<LeaseResource> ReadOrCreateLeaseResource(string name)
             {
                 switch ((HttpStatusCode)e.Status)
                 {
+                    case HttpStatusCode.PreconditionFailed:
```

Comment:
Bug fix, we did not use any pre-condition in our code but the driver implicitly added a pre-condition internally, need to capture this exception.

Reply:
Nice catch